tests/unit/components/status/StatusBar.spec.js

```diff
@@ -41,8 +53,10 @@ describe("StatusWidget.vue", () => {
   ])(
     "When Vuex is mutated to the state %s, Then the status text should update to be: %s",
```

Comment:
if you are mocking a URL, please include that in the `Given` section of a test title---Given that /shutdown is mocked to return status 200, When...

Reply:
Completed.